src/cluster.c

```diff
@@ -1171,6 +1265,8 @@ void clusterDelNode(clusterNode *delnode) {
     dictIterator *di;
     dictEntry *de;
 
+    clusterRemoveNodeFromShard(delnode->shard_id, delnode);
+
     /* 1) Mark slots as unassigned. */
```

Comment:
I suppose all the numbering is not correct anymore.

Reply:
Good catch. Will update the numbering.